crates/voicevox_core/src/engine/open_jtalk.rs

```diff
@@ -55,6 +57,48 @@ impl OpenJtalk {
         Ok(s)
     }
 
+    pub fn load_user_dict(&self, user_dict: &UserDict) -> crate::result::Result<()> {
+        let Some(dict_dir) = &self.dict_dir else {
+            return Err(Error::NotLoadedOpenjtalkDict)
+        };
+
+        let Some(dict_dir) = dict_dir.to_str() else {
+            return Err(Error::NotLoadedOpenjtalkDict);
+                       };
```

Comment:
こういう新しい構文だとRustfmtが対応してなくて仕事を放棄することがあるんですよね...

let-elseの場合、まだ詳しく調べてませんがnightlyのRustfmtだと「実装済み」で利用可能である雰囲気がしています。

あとlet-else使わなくてもこんな感じでいいのでは? と思います(好みの話しかもしれませんが)

```suggestion
        let dict_dir = self
            .dict_dir
            .as_ref()
            .and_then(|dict_dir| dict_dir.to_str())
            .ok_or(Error::NotLoadedOpenjtalkDict)?;

```